src/user.js

```diff
@@ -32,6 +33,34 @@ class User {
       });
     });
   }
+
+  // Recipe to Cook Section //
+  addToRecipeToCook(recipe) {
+    if (!this.recipesToCook.includes(recipe)) {
+      this.recipesToCook.push(recipe)
+    }
+  }
+
+  removeFromRecipeToCook(recipe) {
+    const i = this.recipesToCook.indexOf(recipe);
+    this.recipesToCook.splice(i, 1)
+  }
+
+  filterRecipeToCook(tag) {
+    return this.recipesToCook.filter(recipe => {
+      return recipe.tags.includes(tag);
+    });
+  }
+
+  findRecipeToCook(searchWord) {
```

Comment:
Similarly, should this be findRecipesToCook because it can return multiple recipes via filter?

Reply:
That's a good point, plural seems like the right choice for these types of methods